improver/calibration/rainforest_calibration.py

```diff
@@ -461,124 +467,98 @@ def _calculate_error_probabilities(
                 Cubelist containing the independent feature variables for prediction.
 
         Returns:
-            A cube containing error exceedence probabilities.
+            A cube containing threshold exceedence probabilities.
 
         Raises:
             ValueError:
                 If an unsupported model object is passed. Expects lightgbm Booster, or
                 treelite_runtime Predictor (if treelite dependency is available).
         """
 
-        error_probability_cube = self._prepare_error_probability_cube(forecast_cube)
+        threshold_probability_cube = self._prepare_threshold_probability_cube(
+            forecast_cube, self.model_thresholds
+        )
 
         input_dataset = self._prepare_features_array(feature_cubes)
 
-        forecast_data = forecast_cube.data.ravel()
+        lead_time_hours = forecast_cube.coord("forecast_period").points[0] / (
+            SECONDS_IN_MINUTE * MINUTES_IN_HOUR
+        )
 
         self._evaluate_probabilities(
-            forecast_data,
-            input_dataset,
-            forecast_cube.name(),
-            forecast_cube.units,
-            error_probability_cube.data,
+            input_dataset, lead_time_hours, threshold_probability_cube.data,
         )
 
         # Enforcing monotonicity
-        error_probability_cube.data = self._make_decreasing(error_probability_cube.data)
+        threshold_probability_cube.data = self._make_decreasing(
+            threshold_probability_cube.data
+        )
 
-        return error_probability_cube
+        return threshold_probability_cube
 
     def _get_ensemble_distributions(
-        self, error_CDF: Cube, forecast: Cube, output_thresholds: List[float]
+        self, per_member_CDF: Cube, forecast: Cube, output_thresholds: ndarray
```

Comment:
I realised this after my suggestion, but there is now a mix between ensemble member and ensemble realization. Would it be best to stick with one, that is to map references to ensemble member to ensemble realization? and `per_member_...` to `per_realization_...`? This is not a big deal, so happy to leave as is.

Reply:
changed to `per_realization` everywhere

Reply:
Thanks. I think this makes it clearer and aligns with the cf-conventions for describing the ensemble members.